apps/storybook/src/Tile.stories.tsx

```diff
@@ -197,6 +197,41 @@ Actionable.args = {
   moreOptions: undefined,
 };
 
+export const AnchorLink: Story<TileProps> = (props) => {
+  const {
+    name,
+    description = 'If you click on this stadium, it is going to be selected.',
+    metadata,
+    badge,
+    thumbnail,
+    isNew,
+    isActionable = true,
+    ...rest
+  } = props;
+
+  return (
+    <Tile
+      name={
+        <Tile.Action onClick={() => console.log('clicked')}>{name}</Tile.Action>
+      }
+      description={description}
+      metadata={metadata}
+      badge={badge}
+      thumbnail={thumbnail}
+      isNew={isNew}
+      isActionable={isActionable}
+      {...rest}
+    />
+  );
+};
+AnchorLink.argTypes = { ...Basic.argTypes };
+AnchorLink.args = {
+  ...Basic.args,
+  description:
+    'If you click on this stadium, it is going to open another page.',
+  isActionable: true,
+  moreOptions: undefined,
+};
```

Comment:
Could you make this example actually "open" a new page, even if it's just the same page? In this storybook example, the text says "If you click on this stadium, it is going to open another page.", but it seems broken because nothing happens when it's clicked on. It could be like the links in the story fo `SkipToContentLink` or like in `Anchor - Basic`.

Reply:
Added link to story in docs tab.